pkgs/development/compilers/llvm/17/default.nix

```diff
@@ -150,6 +150,9 @@ in let
           src = ../common/clang/clang-at-least-16-LLVMgold-path.patch;
           libllvmLibdir = "${tools.libllvm.lib}/lib";
         })
+      ] ++ lib.optionals stdenv.targetPlatform.isOpenBSD [
+        # TODO make unconditional
+        ../common/clang/openbsd-no-assume-path-exists.patch
```

Comment:
https://github.com/llvm/llvm-project/pull/92183

Reply:
@RossComputerGuy I suppose maybe when they are merged to master, we can backport to at least 18 and switch to using that for this?

Reply:
@Ericson2314 Sounds good to me.

Reply:
https://github.com/llvm/llvm-project/pull/92293 is a backport PR (that unlike the original I cannot merge). We can cherry-pick instead for extra precision for LLVM 18.

Reply:
Switched to LLVM 18 with these patches.